### PR TITLE
[Feature] Documentation on the `--config` flag when using init

### PR DESCRIPTION
**Pull request by LatentDream**
Close: https://github.com/ploomber/cli/pull/146#discussion_r1787222055

----
📚 Documentation preview 📚: https://ploomber-doc--287.org.readthedocs.build/en/287/

### CODE REVIEW

File: doc/user-guide/cli.md [hunk 1]
```diff
@@ -88,6 +88,14 @@ Then, execute:
 ploomber-cloud deploy
 ```
 
+**Optional Configuration**:
+You can use the `--config` flag with the `ploomber-cloud init --from-existing` command to specify a custom name for the configuration file instead of the default `ploomber-cloud.json`. When using this flag, only the file name will be considered, regardless of any specified path. For example:
```
Comment (neelasha23) [inline doc/user-guide/cli.md:92]: Please add a reference to this [section](https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html#switching-the-configuration-file)

Comment (LatentDream) [reply doc/user-guide/cli.md:92]: ✅ Added https://github.com/ploomber/doc/pull/287/commits/7d7b32d0e79282703e6f7886cc22c2fe7e19cdcf
✅ Fix Warning block rendering https://github.com/ploomber/doc/pull/287/commits/7cc4de51fd63ac9ec661d323459499896333b27e

Comment (neelasha23) [reply doc/user-guide/cli.md:92]: This is not clear: `..and any specified path is ignored.`. Also, I meant in this section you can add a link something like, `Refer to [this section](https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html#switching-the-configuration-file) to learn more about switching configurations. You can look at other files to see how we link sections from the same file.

Comment (LatentDream) [reply doc/user-guide/cli.md]: Just to be sure, is this what you had in mind ? https://github.com/ploomber/doc/pull/287/commits/b525cfdda29b6b7e7c6c4de7505f2baf8da47ecd 
- Precise that --config can but use with a path. not just a name
- Refer to the config section to know more about switching the config file

File: doc/user-guide/cli.md [hunk 2]
```diff
@@ -428,12 +446,14 @@ ploomber-cloud labels --sync
...
-All commands that read or write the config file accept the `--config/-c` argument, for example:
+All commands that read or write the config file accept the `--config/-c` argument. You can use this option to specify both alternative file names and paths, for example:
 
 ```sh
 # create the config file in ploomber-cloud.dev.json
 ploomber-cloud init --config ploomber-cloud.dev.json
 
 # deploy using the dev config
-ploomber-cloud deploy --config ploomber-cloud.dev.json
+ploomber-cloud deploy --config ./config/ploomber-cloud.dev.json
```
Comment (neelasha23) [inline doc/user-guide/cli.md:456]: Why did we add `./config/..`?

Comment (LatentDream) [reply doc/user-guide/cli.md:456]:
```suggestion
ploomber-cloud deploy --config ploomber-cloud.dev.json
```

Comment (LatentDream) [reply doc/user-guide/cli.md]: fix: https://github.com/ploomber/doc/pull/287/commits/39b8ba26e5be7e7a96ae1353dae0b0b36bd29b14

**Review by neelasha23 — APPROVED** [review]

**Review by edublancas — APPROVED** [review]

**Pull request merged**